include/violite.h

```diff
@@ -264,7 +265,7 @@ struct st_VioSSLFd *new_VioSSLConnectorFd(
     const char *key_file, const char *cert_file, const char *ca_file,
     const char *ca_path, const char *cipher, const char *ciphersuites,
     enum enum_ssl_init_error *error, const char *crl_file, const char *crl_path,
-    const long ssl_ctx_flags, const char *server_host);
+    const long ssl_ctx_flags, const char *server_host, bool verify_identity);
```

Comment:
This is split out into a separate argument since before, setting the host implied `verify_identity`. The goal here is though to always send the SNI extension independent of the verify mode, so that's why it's now a separate flag here.